lore/pipelines.py

```diff
@@ -18,6 +18,8 @@
 class TrainTestSplit(object):
     __metaclass__ = ABCMeta
     
+    test_size = 0.1
```

Comment:
Should this be an instance variable with a default in the constructor? I can see different hcild classes in the same project wanting different split sizes.

Reply:
It could, I just wanted to save people the hassle of overloading a constructor.